hijack/templatetags/hijack_tags.py

```diff
@@ -51,7 +51,9 @@ def can_hijack(hijacker, hijacked):
 
 @register.filter
 def is_hijacked(request):
-    return request.session.get('is_hijacked_user', False)
+    return (request is not None and
+            getattr(request, "session", None) is not None and
+            request.session.get('is_hijacked_user', False))
```

Comment:
Why would there be not request, or rather, why would it be `None`? I feel raising an error in case someone passes something that isn't a request to the filter is a good thing, to avoid potential security vulnerabilities.

Reply:
@codingjoe - I went back through the old emails from when we first bumped into this.   We ended up with a cascading chain of 500s triggered by an exception in one of the views.  The original exception would normally render a "something went wrong template" but because of this bug in django-hijack, attempting to render the something went wrong template also caused an exception, and on it went.  The definitive work around at the time was to add this snippet to the template:

`{% if request and request.session and request|is_hijacked %}`

We needed both the checks of request and request.session to prevent the chain of errors and that's why I added it to both places.  With the workaround in place we moved on to other things and didn't dig deeper.  And I thought it would be helpful to do the pull-request since it seems so straightforward to check/fix.  Sorry I can't be of more help.